code-for-canada/project-thundercat
# [eMIB Introduction] Replaced instruction pages with one single intro stub page

# Description

I removed all the instruction pages of the eMIB and replaced it with a single introduction stub page. I also updated the popup box message by creating a new component called 'ConfirmEnterEmib'. That being said, I removed the 'ConfirmStartTest' component from the eMIB, since this component will be used with the _Start Test_ action in a future PR.

## Type of change

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

**When you click on _Start eMIB Sample Test_:**

![image](https://user-images.githubusercontent.com/23021242/56828935-70b2dd80-6830-11e9-8154-9bcba5300c30.png)

**You see this following page:**

![image](https://user-images.githubusercontent.com/23021242/56828972-84f6da80-6830-11e9-90cb-2f40efa64e1d.png)

**If you select _Enter eMIB_:**

![image](https://user-images.githubusercontent.com/23021242/56829009-9a6c0480-6830-11e9-876b-e1235c3eafd4.png)

**If you select then _Enter eMIB_, it brings you to the eMIB test:**

![image](https://user-images.githubusercontent.com/23021242/56829059-bd96b400-6830-11e9-9ef3-dc2b4e9ba585.png)

# Testing

Manual steps to reproduce this functionality:

1.  Select the _Prototype_ from the Main Menu.
2.  Click on _Start the eMIB Sample Test_.
3.  Make sure you can now see one single page.
4.  Then, Click on _Enter eMIB_.
5.  Make sure that you are able to close the popup box and enter the eMIB.

# Checklist

Applicable for all code changes.
